client/src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from "react";
+import {
+  AppBar,
+  Toolbar,
+  IconButton,
+  Button,
+  Box,
+  Hidden,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+import MenuIcon from "@material-ui/icons/Menu";
+import { Link } from "react-router-dom";
+import MobileNavbar from "./mobileNavbar";
+import { UserContext } from "../../context/Context";
+import logo from "../../img/logo.png";
+
+const useStyles = makeStyles(theme => ({
+  root: { flexgrow: 1 },
+  menuButton: { marginRight: "auto", color: "red" },
+  toolbar: {
+    backgroundColor: "transparent",
+    boxShadow: "0 0 0",
+  },
+  toolbarAuth: {
+    backgroundColor: "white",
+  },
+  sitterLink: { color: "black", marginRight: "30px" },
+  logo: {
+    flexGrow: 1,
+    marginRight: "auto",
+    maxWidth: "150px",
+  },
+}));
+
+function Navbar() {
+  const {
+    mobileMenuOpen,
+    setMobileMenuOpen,
+    isAuthenticated,
+    setIsAuthenticated,
+  } = useContext(UserContext);
+  const classes = useStyles();
+
+  return (
+    <AppBar
+      position="fixed"
+      className={isAuthenticated ? classes.toolbarAuth : classes.toolbar}
+    >
+      <Toolbar variant="dense">
+        <Hidden mdUp>
+          <IconButton
+            edge="start"
+            className={classes.menuButton}
+            onClick={() => setMobileMenuOpen(true)}
+            style={{ marginRight: "auto" }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <MobileNavbar open={mobileMenuOpen} />
+        </Hidden>
+        <img src={logo} alt="logo" className={classes.logo} />
+        <Hidden smDown>
+          <Box mr={4} ml={4}>
+            <Link to="*">
+              <Button
+                color="secondary"
+                size="large"
+                className={classes.sitterLink}
+              >
+                Become a Sitter
+              </Button>
+            </Link>
+            <Link to="/login" style={{ textDecoration: "none" }}>
+              <Button color="secondary" variant="outlined" size="large">
+                Login
+              </Button>
+            </Link>
+          </Box>
+          <Link to="/signup" style={{ textDecoration: "none" }}>
+            <Button color="secondary" variant="contained" size="large">
```

Comment:
The color for this button doesn't seem to match the rest of the app. Can you check that? May need to be color primary

Reply:
But this one is according to mocks as well. Are my mocks outdated?

Reply:
The color in the mocks matches the logo (#f04040) according to my eyedropper tool

Reply:
I can see it now too. Barely noticable with naked eye

Reply:
[This](https://chrome.google.com/webstore/detail/eye-dropper/hmdcmlfkchdmnmnmheododdhjedfccka/related?hl=en) is a good tool for checking colors match in cases like this